DesignSafe-CI/portal
# task/DES-2330: Add support for licenses in API

## Overview: ##
Added ability to request a user's matlab and/or ls-dyna licenses via API call

## PR Status: ##


## Related Jira tickets: ##

* [DES-2330](https://jira.tacc.utexas.edu/browse/DES-2330)

## Summary of Changes: ##
- Added views and corresponding URLs to grab a user's matlab/LS-DYNA license

## Testing Steps: ##
*You will need to be a superuser locally to test this*
1. Go to https://designsafe.dev/api/licenses/MATLAB/{your_username}/ and confirm there is a dict showing `{"license": "SERVER matlab.shared.utexas.edu 000000000000 12180\nUSE_SERVER"}`
2. Go to https://designsafe.dev/api/licenses/LSDYNA/{your_username}/ and confirm there is a dict showing `{"license": "license"}`

## UI Photos:

## Notes: ##
